Makefile

```diff
@@ -477,44 +477,50 @@ $(LOCALBIN):
 
 ## Tools
 KUSTOMIZE ?= $(LOCALBIN)/kustomize
-# same as used in kubectl v1.28.x ; see https://github.com/kubernetes-sigs/kustomize/tree/master?tab=readme-ov-file#kubectl-integration
+
+# https://github.com/kubernetes/kubectl/blob/release-1.30/go.mod#L44
+# same as used in kubectl v1.30.x; https://github.com/kubernetes/kubectl/blob/release-1.30/go.mod#L44
```

Comment:
The line you linked to says kustomize v5.0.4, no?
```
sigs.k8s.io/kustomize/kustomize/v5 v5.0.4-0.20230601165947-6ce0bf390ce3
```

Reply:
So apparently [there isn't a kustomize 5.0.4 release](https://github.com/kubernetes-sigs/kustomize/releases?page=4) (there is 5.0.3 and 5.1.0) so I left the version that was already there. I guess I can explain my reasoning in that comment though 😅